internal/host/host.go

```diff
@@ -974,45 +977,60 @@ func (m *Manager) updateValidationsInDB(ctx context.Context, db *gorm.DB, h *mod
 	return hostutil.UpdateHost(logutil.FromContext(ctx, m.log), db, h.InfraEnvID, *h.ID, *h.Status, "validations_info", string(b))
 }
 
-func (m *Manager) AutoAssignRole(ctx context.Context, h *models.Host, db *gorm.DB) error {
-	// select role if needed
+func (m *Manager) AutoAssignRole(ctx context.Context, h *models.Host, db *gorm.DB) (bool, error) {
```

Comment:
It's unclear what this bool signifies

Reply:
I will add a comment on the next PR. It is used for marking that computation of role was done. And it is temporary usage case